others/plot_filedependency.R

```diff
@@ -0,0 +1,51 @@
+# 1. Load required packages (install first if needed)
+if (!requireNamespace("stringr", quietly = TRUE)) install.packages("stringr")
+if (!requireNamespace("igraph",   quietly = TRUE)) install.packages("igraph")
+
+library(stringr)
+library(igraph)
+
+# 2. List all R scripts in your target folder
+files_full <- list.files(
+  path       = "/Users/lauraliu/Library/CloudStorage/Dropbox/projects/sandbox/BDML/simulation",
```

Comment:
This wouldnt work, I'd set the working directory and then use a relative path.
```suggestion
  path       = "../simulation",
```